apps/mobile-wallet/src/screens/Dashboard/DashboardScreen.tsx

```diff
@@ -56,25 +61,25 @@ interface ScreenProps
     BottomBarScrollScreenProps {}
 
 const DashboardScreen = ({ navigation, ...props }: ScreenProps) => {
+  const insets = useSafeAreaInsets()
+  const { t } = useTranslation()
   const theme = useTheme()
+  const { screenScrollY, screenScrollHandler } = useScreenScrollHandler()
+  const currency = useAppSelector((s) => s.settings.currency)
   const totalBalance = useAppSelector(selectTotalBalance)
+  const selectAddessesTokensWorth = useMemo(makeSelectAddressesTokensWorth, [])
+  const balanceInFiat = useAppSelector((s) => selectAddessesTokensWorth(s, addressHashes))
```

Comment:
I think this can be written as following since we want the total amount, to be tested
```suggestion
  const balanceInFiat = useAppSelector(selectAddessesTokensWorth)
```

Reply:
Done, it works.